trace/trace.go

```diff
@@ -83,6 +86,31 @@ func (t *Trace) finish() {
 	t.End = time.Now()
 }
 
+// (Experimental)
+// Enabled sets tracing to enabled.
```

Comment:
This will actually trip the lint rules from the other PR. `Enable` needs to be the first word in the comment to satisfy the linter.